node_modules/oae-core/filepreview/filepreview.html

```diff
@@ -54,6 +54,32 @@
     </div>
 --></div>
 
+<!-- VIDEO PREVIEW -->
+<div id="filepreview-video-template"><!--
+    <div id="filepreview-video-container" class="text-center">
+        {if content.previews && content.previews.video}
+            {var url = content.previews.video.replace('remote:', '')}
+            {var mime = 'video/mp4'}
+        {else}
+            {var url = content.downloadPath}
```

Comment:
In what scenario do we want to show a video player without there being a generated preview for it ?